content/release-10-17-0/applications-10-17-0-bundle/applications-10-17-0-286.md

```diff
@@ -0,0 +1,49 @@
+---
+weight: 48
+title: 10.17.0.286
+layout: bundle
+---
+
+<!--10.17.0.269 - 10.17.0.286-->
+
+
+### Fixes
+
+<table>
+<colgroup>
+<col style="width: 15%;">
+<col style="width:50%;">
+<col style="width: 10%;">
+<col style="width: 12%;">
+<col style="width: 13%;">
+</colgroup>
+<thead><tr>
+<th>
+Product area</th>
+<th>
+Description</th>
+<th>
+Issue</th>
+<th>
+Build version</th>
+<th>Build comp.</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>Cockpit</td>
+<td>The KPI widget shows the correct previous value when hovering over the trend indicator, if no real-time updates have been received yet.</td>
+<td>MTM-55095</td>
+<td>10.17.0.280</td>
+<td>UI</td>
+</tr>
+
+<tr>
+<td>Web SDK</td>
+<td>In the data explorer, when deleting the min/max value of a data point, the Y-axis displayed "-1" and "1" instead of determining the maximum and minimum value based on the data. This has been fixed and the Y-axis shows the min/max value of the data again.</td>

```

Comment:
```suggestion
<td>In the data explorer, when deleting the minimum or maximum value of a data point, the Y-axis displayed "-1" and "1" instead of determining the maximum and minimum value based on the data. This has been fixed and the Y-axis shows the minimum and maximum values of the data again.</td>

```